frontend/workflows/projectCatalog/src/details/info/repositoryRow.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Grid } from "@clutch-sh/core";
+import type { IconProp } from "@fortawesome/fontawesome-svg-core";
+import { faGithub } from "@fortawesome/free-brands-svg-icons";
+import { faCode, faCodeBranch } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import { LinkText } from "../helpers";
+
+interface ProjectRequests {
+  number: number;
+  url?: string;
+}
+
+const RepositoryRow = ({ repo }: { repo: string }) => {
+  const [name, setName] = React.useState<string>("");
+  const [url, setUrl] = React.useState<string>();
+  const [icon, setIcon] = React.useState<IconProp>(faCode);
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [requests, setRequests] = React.useState<ProjectRequests>();
+
+  React.useEffect(() => {
+    if (repo) {
+      const [base, splitProject] = repo.split(":");
+      const project = splitProject.replace(".git", "");
+      const [, manager] = base.split("@");
+
+      if (project.indexOf("/") > 0) {
+        setName(project.split("/").pop() || project);
+      } else {
+        setName(project);
+      }
+
+      if (manager) {
+        setUrl(`https://${manager}/${project}`);
+
+        switch (manager.toLowerCase()) {
+          case "github.com":
+            setIcon(faGithub);
+            break;
+          default:
+            setIcon(faCode);
+        }
+
+        // TODO (jslaughter): fetch open PR's count
+        // setRequests({
+        //   number: 0,
+        //   url: `https://${manager}/${project}/pulls`,
+        // });
```

Comment:
good callout. We will likely need to do this on the server since the remote could be private and require authentication

Reply:
Yeah I was looking to see if we had anything setup but seems not currently. It stays hidden until we provide something though so it wont affect anything at least.